apps/wallet-mobile/src/features/Swap/common/SelectPool/SelectPoolFromList/SelectPoolFromList.tsx

```diff
@@ -7,6 +7,7 @@ import LinearGradient from 'react-native-linear-gradient'
 import {Spacer} from '../../../../../components'
 import {useMetrics} from '../../../../../metrics/metricsManager'
 import {COLORS} from '../../../../../theme'
+import {useSwapTouched} from '../../../useCases/StartSwapScreen/CreateOrder/TouchedContext'
```

Comment:
if is used by more than 1 move to `common`. (-> `TouchedContext` -> `common/SwapFormProvider`)